package.json

```diff
@@ -12,6 +12,8 @@
         "build-docs": "cd src/docs/website && npm run build",
         "build-editor": "cd src/editor && npm run build",
         "build-server": "parcel build src/index.html",
+        "build-share":
```

Comment:
Consider splitting this into three scripts.

Reply:
Ok fixed it, thank you